src/templates/ObservabilityPackDetails.js

```diff
@@ -32,6 +36,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
       packId: pack.id,
     }
   );
+
+  const getPackUrl = () => {
+    const baseUrl =
+      'https://github.com/newrelic/newrelic-observability-packs/tree/main/packs';
+
+    if (!pack.logoUrl) {
+      return baseUrl;
+    }
+
+    /**
+     * logoUrl looks something like: 'https://raw.githubusercontent.com/newrelic/newrelic-observability-packs/v0.8.2/packs/couchbase/logo.svg'
+     */
+    const [packName, ..._] = pack.logoUrl.split('/').slice(-2);
```

Comment:
You don't actually need to destructure the remaining arguments. If all you want is the first item you can just do this:

```suggestion
    const [packName] = pack.logoUrl.split('/').slice(-2);
```

Reply:
Furthermore, if you're just trying to get the packname, you shouldn't need to destructure anything: just supply a second parameter to [`Array.slice`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice):
```suggestion
    const packName = pack.logoUrl.split('/').slice(-2, -1);
```

Reply:
changed to use url, and shorter destructure